.github/scripts/build-images.sh

```diff
@@ -42,6 +45,8 @@ if (grep -q "${DF_PATH#./}" <<<$modified_files) || # Rebuild the image if any fi
         echo "No extra arch is found, skipping install QEMU."
     fi
     
+    echo "MODIFIED=true" >> $GITHUB_ENV
```

Comment:
I'm not sure I understood what you meant, do you mean in other branches (other than master) we don't push images to DockerHub and because of that the follow-up images (aka children) won't get the latest image in their rebuilding process? If that's what you mean that is correct. But there is no other way of doing this unfortunately. We have to build the parent images manually and push them in such cases. Do you have a better idea?

Reply:
No better idea :sweat_smile:

Reply:
What about keeping and pushing to cloudsuitetest for such cases to make building children images meaningful?

Reply:
The problem with that is that it requires modifying the dockerfiles to change the `FROM` statement to use another repo. It's definitely possible though. We can use the same `is_parent_modified` flag and if that's true we should update the dockerfile to use a different repo and build the image via that repo.
Let's now worry about that for now but have this as an unresolved issue and you can get back to it when we are done with everything here.

Reply:
I also pushed an update for this suggestion.